tools/config/go-templates/model_simple.mustache

```diff
@@ -90,14 +90,16 @@ func New{{classname}}WithDefaults() *{{classname}} {
 
 {{#vars}}
 {{#required}}
-// Get{{name}} returns the {{name}} field value
+{{! X-XGEN-MODIFIED }}
```

Comment:
What is this for?

Reply:
Just to keep track of what we have modified in the templates, so we can easily merge the templates from upstream with our modifications.